docs/runtimes/application.ipynb

```diff
@@ -135,6 +135,40 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Other API Gateway configurations\n",
+    "\n",
+    "There are also other parameters that can be configured when creating an API gateway:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "application.create_api_gateway(\n",
+    "    # The name of the API gateway\n",
+    "    name=\"my-api-gateway\",\n",
+    "    # Optional path of the API gateway, default value is \"/\". Note that given path should be supported by deployed application\n",
```

Comment:
# Optional path of the API gateway, default value is \"/\". The given path should be supported by the deployed application\n",